src/color-chart/color-chart.js

```diff
@@ -101,7 +101,12 @@ const Self = class ColorChart extends ColorElement {
 		}
 
 		if (isFinite(min) && isFinite(max)) {
-			const axisData = getAxis(min, max, 10);
+			const axisData = getAxis({
+				min,
+				max,
+				initialSteps: 10,
+				force: { min: !isNaN(this[`${axis}Min`]), max: !isNaN(this[`${axis}Max`]) }, // follow the provided min/max values, if any
```

Comment:
This is a bit of an obscure condition, but also we check for these earlier (L94 and L100), so we can just set flags there. Or even just a `let force = {min: true, max: true}` and ovverride it in these conditionals, then you don't even need to pass a value here.